js/components/AddExpertEventFeedItem.jsx

```diff
@@ -28,8 +28,10 @@ var AddExpertEventFeedItem = React.createClass({
       url: url,
       type: 'GET',
       success: function(user) {
-        if (user.picture && user.picture.url) {
-          this.setState({ img: user.picture.url });
+        if (this.isMounted()) {
```

Comment:
These checks prevent invariant violations from clicking through the feed too fast